### ISSUE TITLE
Entities in email alerts

### ISSUE THREAD

**Issue by duncanparkes** [mysociety/theyworkforyou#88]
Email alerts have started to have &amp;#8217;s where they should have apostrophes.

**Comment by MyfanwyNixon** [comment]
A user writes:

> Please see the alert email below.  It contains the `"&#8220"` and `"&#8221"` entities instead of the quotation symbols.

> Not a massive bug, but maybe you might look at whether your source data
has changed coding.

**Comment by garethrees** [comment]
I just noticed this:

```
Sexual Offences (Pardons etc) Bill (21 Oct 2016)
https://www.theyworkforyou.com/debates/?id=2016-10-21a.1071.4&s=speaker%3A25329#g1077.2
Craig Williams: I commend the hon. Gentleman for the tone he is adopting
today, but he said &#8220;English only&#8221;. Speaking as a Welsh MP, I
point out that it is England and Wales. I would like to remind him of
```

**Comment by dracos** [comment]
Took a while! But I believe this was fixed by eafb7eae and b900b14d in order to fix #1386.
